ERCS/eip-draft-erc-bts.md

```diff
@@ -0,0 +1,139 @@
+---
```

Comment:
```suggestion
---
eip: 7621
```
Assigning next sequential EIP/ERC/RIP number.
Numbering changed to sequential from 7500 and is no longer the PR number.

Please also update the filename.